docs/userGuide/glossary.md

```diff
@@ -34,7 +34,16 @@ MarkBind is able to take Markdown files (`.md`) and HTML files (`.html`) as sour
 
 <span id="site-navigation-menu">
 
-_Site Navigation Menu_ (_siteNav_ for short) is a fixed menu on the left side of your page contents, that shows a road map of the main pages of your site.
+_Site Navigation Menu_ (_siteNav_ for short) is a fixed menu on the **left** side of your page contents, that shows a road map of the main pages of your site.
 </span>
 
-</div>
\ No newline at end of file
+</div>
+
+#### Page Navigation Menu
```

Comment:
The indentation is off with the rest of the entries.

<img src="https://user-images.githubusercontent.com/3168908/50958982-dcc93a00-14fc-11e9-9612-f4a38e208a22.png" width="400" />